README.md

```diff
@@ -121,7 +121,7 @@ Same as `run`, a set of options can be set. By default, this method does not sen
 - `limit_count`: [Integer] Mutates `circuit_switches.report_limit_count` whose value defined in schema is 10 by default. (default: nil)  
   Can't be set to 0 when `stop_report_if_reach_limit` is true.
 
-To know about report is executed or not, you can get through `report?`.  
+To know if `report` has already been executed or not, you can get through `reported?`.  
```

Comment:
Or would this be perhaps better when calling it means executing it and there is no middle?

```suggestion
To know if `report` has already been called or not, you can get through `reported?`.  
```

Reply:
Hmm, `report` will be always called, and we want to know `report` executes report or not through `reported?`. So I think executed is suitable.

Reply:
I was slightly anxious that the word came directly from `execute_report` which is an internal api and users need not be  aware of. But this is actually a style of less importance and I'm alright with this. Thanks!